1_fetch.yml

```diff
@@ -4,18 +4,21 @@ include:
   - lib.yml
 
 packages:
+  - dataRetrieval
   - httr
 
 file_extensions:
   - ind
 
 sources:
+  - 1_fetch/src/fetch_streamflow.R
   - 1_fetch/src/fetch_storm_track.R
 
 targets:
 
   1_fetch:
     depends:
+      - 1_fetch/out/streamflow.rds.ind
       - 1_fetch/out/storm_track.zip
```

Comment:
oops, good call - the storm_track depends item should also point to the .ind version of that file.